application_form/services/queue.py

```diff
@@ -37,8 +38,9 @@ def add_application_to_queues(
                     apartment_uuid, application_apartment
                 )
                 list_position = queue_position
-                # Need to shift both list position and queue position
-                _shift_positions(apartment_uuid, queue_position)
+                with transaction.atomic():
```

Comment:
Not sure if this is needed, there is already a transaction started in `lock_table()` ?

Reply:
Yes, this is not needed